ShareMyOffer/src/main/java/com/neeraja/ShareMyOffer/dto/LoginDTO.java

```diff
@@ -1,5 +1,37 @@
 package com.neeraja.ShareMyOffer.dto;
 
+import org.springframework.stereotype.Component;
+
+@Component
 public class LoginDTO {
+	
+	private String username;
+	
+	private String password;
+	
+	public LoginDTO() {
+		
+	}
+	
+	public LoginDTO(String username,String password) {
+		this.username = username;
+		this.password = password;
+	}
+	
+	public String getUsername() {
```

Comment:
no lombok yet

Reply:
Yeah there was a problem with eclipse but now I switched to IntelliJ and updated it to Lombok